.github/ISSUE_TEMPLATE/release.md

```diff
@@ -11,11 +11,14 @@ Release vX.X.X
 
 ## Checklist
 
-- [ ] Update metadata & clusterctl-settings.
-- [ ] Update version in e2e config.
+- [ ] (optional) Update metadata & clusterctl-settings.
+- [ ] (optional) Update version in e2e config.
 - [ ] Update docs (compatibility table; usage etc).
-- [ ] Create release branch `release-X.Y`
+- [ ] (optional) Create release branch `release-X.Y`
+- [ ] Cherry-pick all relevant changes into the release branch.
 - [ ] Create tag `vX.Y.Z`.
 - [ ] Update the created draft release to include things like breaking changes or important notes.
-- [ ] Update the draft release notes with generated release notes from `sigs.k8s.io/kubebuilder-release-tools/notes`
+- [ ] Update the draft release notes with generated release notes from `sigs.k8s.io/cluster-api/hack/tools/release/notes`
```

Comment:
On line 20c you're saying update release notes with important notes and breaking changes.

Which step should be first. 
I believe generating the release notes should be first.